immer/detail/rbts/node.hpp

```diff
@@ -238,7 +238,7 @@ struct node
     {
         assert(n <= branches<B>);
         auto mp = heap::allocate(sizeof_inner_r_n(n));
-        auto mr = (void*){};
+        auto mr = static_cast<void*>(nullptr);
```

Comment:
Ahá! Good catch! I am wondering whether there might be a more lightweight syntax for AAA pointer declarations... maybe this is ok for now (mutable pointer variables of this kind should not occur often anywayz.)